limatheusm/swift-twitter
# add login screen

#### Issue Summary
- User authentication with OAuth
- Manager User Session

#### Implementation Summary
- LoginScene: logging in using Twitter OAuth
- SplashScene: verify with the user is logged or if the session is still valid and redirect to correct flow
- RootViewController: redirect between Login and Search scenes
- Clean Architecture

#### How to Test
1. Open the APP
2. Login with Twitter OAuth
3. When the user returns to the APP, the user will be redirected to main screen
4. Close the APP
5. After SplashScreen, the APP should redirect to main screen
